packages/terra-i18n/docs/README.md

```diff
@@ -28,6 +53,24 @@ Note that the state of the object needs to contain keys as follows for the i18nL
 }
 ```
 
+Additionally, do not render translated components until areTranslationsLoaded is true.
```

Comment:
Agreed.  I like the idea of explaining what we need to do in the context of an example.